frontend/apps/web/components/ListBox/ListBox.tsx

```diff
@@ -39,16 +39,19 @@ export default function ListBox<TData>(props: Props<TData>): ReactElement {
   return (
     <div
       className={cn(
-        'h-[164px] relative w-full rounded-md border border-gray-300 dark:border-gray-700 ',
+        'h-[164px]  rounded-md border border-gray-300 dark:border-gray-700 w-[350px]',
         tableContainerClassName,
         rows.length > 0 && 'overflow-auto'
       )}
       ref={tableContainerRef}
     >
-      <StickyHeaderTable>
-        <TableHeader className="bg-gray-100 dark:bg-gray-800 sticky top-0 z-10 grid rounded-md">
+      <StickyHeaderTable className="w-full">
```

Comment:
This is redundant. The `<StickyHeaderTable />` is just a wrapper around `<table>` and it applies `w-full`